cde-core/resource/resources/custom/components/exportButton/component.xml

```diff
@@ -90,6 +91,20 @@
             <Version>1.0</Version>
           </Header>
         </DesignerProperty>
+        <DesignerProperty>
+          <Header>
+            <Name>exportPage</Name>
+            <Parent>BaseProperty</Parent>
+            <DefaultValue>false</DefaultValue>
```

Comment:
@pamval @afrjorge making this default to **false** will actually change a legacy default ( a bug ok, but still the default behaviour ), so we should probably keep this defaulted to **true** and have this as a Basic property so its easy to spot and change the behaviour for whoever wants to. What do you think